### ISSUE TITLE
Enable pylint check W0212 (protected access)

### ISSUE THREAD

**Issue by goodfeli** [cleverhans-lab/cleverhans#609]

**Comment by catherio** [comment]
This error only triggers in two files:
* utils_keras.py, to access `_inbound_nodes` and `_op`, as part of some graph surgery to find the logits
* nips17_adversarial_competition/eval_infra/code/eval_lib/tests/classification_results_test.py, while manipulating a `CompetitionSubmissions` object as a test object

These both seem basically OK to me. Is it alright to enable this check by manually skipping the pylint checks on those lines?

**Comment by jonasguan** [comment]
Closing as we upgrade to version 4.
